src/scripts/main.js

```diff
@@ -1,5 +1,34 @@
 'use strict';
 
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
 document.addEventListener('click', (e) => {
-  // write code here
+  const wallCoords = wall.getBoundingClientRect();
+  const spiderCoords = spider.getBoundingClientRect();
+  const border = parseInt(
+    window.getComputedStyle(wall).borderWidth.split('px').join(''),
+  );
+  let spiderY =
+    e.clientY - wallCoords.top - parseInt(spider.offsetWidth) / 2 - border;
```

Comment:
The calculation of `spiderY` uses `spider.offsetWidth`, which might be incorrect if you intended to use `spider.offsetHeight` for vertical positioning. Verify that this is the intended behavior.